closure/goog/dom/forms_test.js

```diff
@@ -417,6 +417,26 @@ function testSetValueInput() {
   assertEquals('', goog.dom.forms.getValue(el));
 }
 
+function testGetValueMeter() {
+  var min = 0;
+  var max = 3;
+  var val = 2.3;
+  var el = goog.dom.createDom('meter', {'min': min, 'max': max, 'value': val});
```

Comment:
Let's inline these values rather than pulling them out as variables.  It should make them a lot easier to read.